cmd/svcat/command/command.go

```diff
@@ -17,15 +17,14 @@ limitations under the License.
 package command
 
 import (
+	"fmt"
 	"strings"
 	"unicode"
 
-	"fmt"
-
 	"github.com/spf13/cobra"
 )
 
-// Command represents an svcat command.
```

Comment:
an SVCAT is actually correct. See https://www.languageediting.com/how-to-choose-the-indefinite-article-a-or-an-before-acronyms/

Reply:
I'm still not quite sure if it should be a `an` or `a`, but I'll keep it as it was (`an`).